ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -93,6 +93,16 @@ nnfw_open (const GstTensorFilterProperties * prop, void **private_data)
     goto unalloc_exit;
   }
 
+  /**
+   * @todo: remove after fixes to nnfw package for aarch64
+   */
+  status = nnfw_set_available_backends(pdata->session, NNFW_DEFAULT_BACKEND);
+  if (status != NNFW_STATUS_NO_ERROR) {
+    err = -EINVAL;
+    g_printerr ("Cannot set nnfw-runtime backend to %s", NNFW_DEFAULT_BACKEND);
+    goto unalloc_exit;
+  }
```

Comment:
Anyway, how are you going to handle PROP_ACCELERATOR?

Reply:
This will be handled in a different PR (hopefully tomorrow), where PROP_ACCELERATOR will be used to set the backend (like in https://github.com/nnsuite/nnstreamer/pull/1847).
As the comment says, this is just a temporary fix to make aarch64, to be removed after nnfw package is fixed.